internal/services/applicationinsights/application_insights_standard_webtests_resource.go

```diff
@@ -151,8 +151,9 @@ func (ApplicationInsightsStandardWebTestResource) Arguments() map[string]*plugin
 					// },
 
 					"ssl_cert_remaining_lifetime": {
-						Type:     pluginsdk.TypeInt,
-						Optional: true,
+						Type:         pluginsdk.TypeInt,
+						Optional:     true,
+						ValidateFunc: validation.IntBetween(1, 365),
```

Comment:
We should add this validation to the docs

Reply:
Done 🎉